template.distillery/PROJECT_NAME.opam

```diff
@@ -4,7 +4,7 @@ version: "0.1"
 synopsis: "Pseudo package for defining project dependencies"
 
 depends: [
-  "eliom" {>= "9.0.0" & < "10.0.0"}
+  "eliom" {>= "9.3.0" & < "10.0.0"}
```

Comment:
Where is eliom.9.3.0 ?

Reply:
https://github.com/ocsigen/eliom/pull/732